dateparser/date.py

```diff
@@ -326,14 +335,15 @@ def get_date_data(self, date_string, date_formats=None):
 
         for language in self.language_detector.iterate_applicable_languages(
                 date_string, modify=True):
-            parsed_date = _DateLanguageParser.parse(language, date_string, date_formats)
+            parsed_date = _DateLanguageParser.parse(
+                language, date_string, date_formats, settings=self._settings)
             if parsed_date:
                 return parsed_date
         else:
             return {'date_obj': None, 'period': 'day'}
 
     @classmethod
-    def _get_language_loader(cls):
-        if not cls.language_loader:
-            cls.language_loader = LanguageDataLoader()
-        return cls.language_loader
+    def _get_language_loader(cls, settings):
+        if any([not cls.language_loaders, settings not in cls.language_loaders]):
```

Comment:
Since `language_loaders` is `{}` by default, I think we can just use second condition here :)